connectors/sources/outlook.py

```diff
@@ -831,6 +834,7 @@ async def download_func(self, content):
         yield content
 
     async def _fetch_attachments(self, attachment_type, outlook_object, timezone):
+        self._logger.debug(f"Fetching attachments for {attachment_type}")
         for attachment in outlook_object.attachments:
```

Comment:
Does this line make a call to Outlook Server?

Reply:
No, we get all the attachments in `outlook_object` i.e. from mail object, task object and calendar object. It does not call an extra call to server.